python/lsst/dax/obscore_generator/combine_obscore_csv.py

```diff
@@ -32,7 +34,7 @@ def exec_combine_obscore_csv(out_dir):
     file_pattern = f"{out_dir}/gen_obscore_"
     src_filenames = [i for i in glob.glob(f"{file_pattern}*")]
     print(f"Combining source files: {src_filenames}")
```

Comment:
We will need to convert these prints to log messages.

Reply:
I think I'm going to drop these obsolete files, I probably won't need them anymore after our discussion at the meeting today.